package.json

```diff
@@ -6,7 +6,7 @@
   "main": "index.js",
   "scripts": {
     "lint": "npm run lint:js && npm run lint:scss",
-    "lint:js": "prettier -c --loglevel warn **/*.js",
+    "lint:js": "eslint --ext=.js,.mjs .",
```

Comment:
@codehag, I don't know if you want to lint the JSON or not, but if so, would you like it done in a separate package run script or should I add the extension here?